src/test/java/org/molgenis/vcf/report/mapper/HtsJdkToSamplesMapperTest.java

```diff
@@ -12,14 +14,15 @@
 import org.junit.jupiter.api.Test;
 import org.molgenis.vcf.report.model.Items;
 import org.molgenis.vcf.report.model.Sample;
+import org.phenopackets.schema.v1.core.Pedigree.Person;
 
 class HtsJdkToSamplesMapperTest {
```

Comment:
add ```@ExtendWith(MockitoExtension.class)```?

Reply:
outdated, but not marked as such due to the fix happening on another line